docs/support/tutorials/gis/eo_guide.md

```diff
@@ -39,221 +54,166 @@ What to consider when chosing data:
 
-[**Julia**](../../../apps/julia.md)
+[**Matlab**](../../../apps/matlab.md) - you can run Matlab jobs on Puhti conveniently from your own computers Matlab installation.
 
-* Julia is provided via the `Julia` module, it does not include any packages, but they can be installed by the user. 
-* [JuliaGeo](https://github.com/JuliaGeo) provides an overview of packages for geospatial data.
-
-[**Matlab**](../../../apps/matlab.md)
-
-* You can run Matlab jobs on Puhti conveniently from your own computers Matlab installation.
-
-[**Orfeo Toolbox (OTB)**](../../../apps/otb.md)
-
-* Offers a wide variety of applications from ortho-rectification or pansharpening, all the way to classification, SAR processing, and much more.
-* Orfeo Toolbox is available as CLI, GUI and via Python interface.
+[**Orfeo Toolbox (OTB)**](../../../apps/otb.md) - offers a wide variety of applications from ortho-rectification or pansharpening, all the way to classification, SAR processing, and much more. Orfeo Toolbox is available as CLI, GUI and via Python interface.
 
 
 [**Python**](../../../apps/python.md)
 
-* The [geoconda module](../../../apps/geoconda.md) provides many useful Python packages for raster data processing and analysis:
+* The [geoconda module](../../../apps/geoconda.md) provides many useful Python packages for raster data processing and analysis, inc best tools for working with STAC catalogues.
+* [Machine learning modules](../../../apps/index.md#data-analytics-and-machine-learning) provide all common machine learning frameworks, also for deep learning.
 
-    * py6s: Python interface to the Second Simulation of the Satellite Signal in the Solar Spectrum (6S) atmospheric Radiative Transfer Model.
-    * rasterio: access to geospatial raster data.
-    * rasterstats: summarizing geospatial raster datasets based on vector geometries.
-    * sentinelsat: downloading Sentinel images
-    * scimage: algorithms for image processing.
-    * stackstac: STAC data to xarray
-    * xarray: working with multidimensional raster data.
 
-* [Geospatial Python examples for Puhti](https://github.com/csc-training/geocomputing/tree/master/python). See also raster lesson of [CSC version of Python GIS course material](https://github.com/csc-training/notebooks) for further examples.
+[**QGIS**](../../../apps/qgis.md) - is a very widely used GUI for working with spatial data, it has also limited multispectral image processing capabilities.  GUI with batch processing possibility and Python interface. Used for example for visualization, map algebra and other raster processing. Many plug-ins available, for EO data processing, check out the [QGIS Semi-automatic classification plugin](https://fromgistors.blogspot.com/p/semi-automatic-classification-plugin.html).
```

Comment:
```suggestion
[**QGIS**](../../../apps/qgis.md) - general open source GIS (Geographic Information System) for working with spatial data with limited multispectral image processing capabilities.  GUI with batch processing possibility and Python interface. Used for example for visualization, map algebra and other raster processing. Many plug-ins available; for EO data processing, check out the [QGIS Semi-automatic classification plugin](https://fromgistors.blogspot.com/p/semi-automatic-classification-plugin.html).
```

Reply:
general open source GIS - even if that is the origin of the name, then it is very bad description, would not use it.

Reply:
```suggestion
[**QGIS**](../../../apps/qgis.md) - open source tool with GUI for working with spatial data including limited multispectral image processing capabilities.  GUI with batch processing possibility and Python interface. Used for example for visualization, map algebra and other raster processing. Many plug-ins available, for EO data processing, check out the [QGIS Semi-automatic classification plugin](https://fromgistors.blogspot.com/p/semi-automatic-classification-plugin.html).
```